singer/transform.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, errors):
             msg = "Errors during transform\n\t{}".format("\n\t".join(estrs))
             msg += "\n\n\nErrors during transform: [{}]".format(", ".join(estrs))
 
+        # pylint: disable=super-with-arguments
         super(SchemaMismatch, self).__init__(msg)
```

Comment:
The pylint complaint here is valid, just need to remove the arguments from super to become `super().__init__..`

Reply:
thank you, fixed by 0f36fa9e9ab18d89cabbd4a9612f8b54c271f36b